src/model/model_json.nit

```diff
@@ -166,6 +166,7 @@ end
 redef class MClassDef
 	redef fun json do
 		var obj = super
+		obj["mdoc"] = mdoc
```

Comment:
Maybe, the json should only and always save the real doc. So change https://github.com/jcbrinfo/nit/blob/ed3ede5205d0fe188824527e7ed83cab26cd795f/src/model/model_json.nit#L61 instead
@Morriar what is your opinion?